src/ports/postgres/cmake/PostgreSQLUtils.cmake

```diff
@@ -82,8 +82,7 @@ function(determine_target_versions OUT_VERSIONS)
                     set(VERSION "4.3ORCA")
                 endif()
             elseif(${PORT_UC} STREQUAL "POSTGRESQL" AND
-                    (${${PORT_UC}_VERSION_MAJOR} EQUAL 10 OR
-                    ${${PORT_UC}_VERSION_PATCH} GREATER 10))
```

Comment:
The intention was to check for `...VERSION_MAJOR >= 10`. Hence the two checks - there was, however a typo with the second version input as `...PATCH` instead of `...MAJOR`. The correction should be to replace the `_PATCH` with `_MAJOR`.

Reply:
@d Do you mind if I merge this with the minor change (as described above)?

Reply:
oops now I understand why the `GREATER` check was there! Yeah go ahead!